### DISCUSSION TITLE
Welcome to eslint-plugin-wxml Discussions!

### DISCUSSION THREAD

**Discussion by iChenLei** [wxmlfile/eslint-plugin-wxml#31] [Announcements]
<p align="center"><a href="https://vuejs.org" target="_blank" rel="noopener noreferrer"><img width="150" src="https://funimg.pddpic.com/mobile_piggy/4d0f5a17-574b-4fbc-aee1-1b0cbb1c46dd.png.slim.c1.png" alt="Vue logo"></a></p>

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Rocking77** [comment]
how to disable specific rule in single wxml file ？

**Reply by iChenLei** [reply]
Sorry for this case, not support __`disable inline comment`__ like `<!-- eslint-disable-next-line -->` yet.

**Reply by iChenLei** [reply]
Do you have any suggestions about it ?

**Reply by Rocking77** [reply]
I have no idea，but maybe you can refer to the apply-disable-directives.js file in eslint

**Reply by Rocking77** [reply]
Thank you for your contribution anyway ！
